### DISCUSSION TITLE
Do we link to authors like we link to, say, email users via mailto?

### DISCUSSION THREAD

**Discussion by oldrup** [Automattic/wordpress-activitypub#388] [General]
Now what have we got here? A fresh 1.0 release? I'm going to [try that out](https://oldrup.dk/en/experiment-activitypub-for-wordpress/).

Stupid question, maybe. But do we link to authors in some way? Like we link to an email using `mailto:user@example.com` - and clicking that link, would open the default email application. Is there as similar `fediverse:@author@instance.net`?  Do I make sense?

**Comment by pfefferle** [comment]
Hey @oldrup 👋

good reminder, that I should finally/officially release and upload 1.0.0 to WP.org!

To your question: ActivityPub does not "know" about WebFinger (the email like identifier: usernam@domain.tld), this is something that was introduced by platforms of the fediverse and Mastodon is even requiring it. So in the ActivityPub world, you would simply link to their profile URL, because this is normally the Identifier that you would use to follow someone.

For the WebFinger Identifier, a new URI scheme was introduced: `acct:` https://www.rfc-editor.org/rfc/rfc7565.html

Sadly there is no actual protocol handler that can handle this identifier properly (no app opens if you click such links).

But there is an interesting discussion about linking to profiles and a simple way to start the follow process: https://codeberg.org/fediverse/fediverse-ideas/issues/1

I hope this helps :)

**Reply by oldrup** [reply]
Interesting thread, thank you for pointing me in the right direction 👍 

Yaah, I think you should take the plunge, and release the 1.0 of your fine plugin. Regardless of us users always wanting more (features), it seems pretty solid at what it does. I was surprised, how easy it was to configure and get up and running, even for a feedinewb like me ;)
